<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>733</width>
    <height>497</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QTableWidget" name="tableWidget">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>10</y>
     <width>311</width>
     <height>391</height>
    </rect>
   </property>
   <property name="rowCount">
    <number>57</number>
   </property>
   <property name="columnCount">
    <number>4</number>
   </property>
   <attribute name="horizontalHeaderDefaultSectionSize">
    <number>148</number>
   </attribute>
   <row>
    <property name="text">
     <string>0</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>1</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>2</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>3</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>4</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>5</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>6</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>7</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>8</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>9</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>10</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>11</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>12</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>13</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>14</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>15</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>16</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>17</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>18</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>19</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>20</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>21</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>22</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>23</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>24</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>25</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>26</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>27</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>28</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>29</string>
    </property>
   </row>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <column>
    <property name="text">
     <string>Name</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Volume</string>
    </property>
   </column>
   <column/>
   <column/>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>10</y>
     <width>167</width>
     <height>398</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QPushButton" name="pushButton_fr">
      <property name="text">
       <string>reset</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_fs">
      <property name="text">
       <string>sent</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="checkBox_f0">
      <property name="text">
       <string>Motor</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="checkBox_f1">
      <property name="text">
       <string>StopOverCurrent</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="checkBox_f2">
      <property name="text">
       <string>StopIfD1</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="checkBox_f3">
      <property name="text">
       <string>StopIfD2</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="checkBox_f4">
      <property name="text">
       <string>StopIfD3</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="checkBox_f5">
      <property name="text">
       <string>StopIfNumberTurns</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="checkBox_f6">
      <property name="text">
       <string>D1</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="checkBox_f7">
      <property name="text">
       <string>D2</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="checkBox_f8">
      <property name="text">
       <string>D3</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="checkBox_f9">
      <property name="text">
       <string>Reset</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="checkBox_f10">
      <property name="text">
       <string>Direction</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="checkBox_f11">
      <property name="text">
       <string>NoTurns</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>10</y>
     <width>167</width>
     <height>396</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QPushButton" name="pushButton_kr">
      <property name="text">
       <string>reset</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_ks">
      <property name="text">
       <string>sent</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="checkBox_k0">
      <property name="text">
       <string>stop</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="checkBox_k1">
      <property name="text">
       <string>ForvardPUSK</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="checkBox_k2">
      <property name="text">
       <string>BackPUSK</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="checkBox_k3">
      <property name="text">
       <string>StopIfD1</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="checkBox_k4">
      <property name="text">
       <string>StopIfD2</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="checkBox_k5">
      <property name="text">
       <string>StopIfD3</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="checkBox_k6">
      <property name="text">
       <string>StopIfNumberTurns</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="checkBox_k7">
      <property name="text">
       <string>Izm_DS18B20</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="checkBox_k8">
      <property name="text">
       <string>Izm_Sonar</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="checkBox_k9">
      <property name="text">
       <string>OffPID</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="checkBox_k10">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="checkBox_k11">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QCheckBox" name="checkBox_k12">
   <property name="geometry">
    <rect>
     <x>-30</x>
     <y>360</y>
     <width>165</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBox_adr">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>440</y>
     <width>91</width>
     <height>27</height>
    </rect>
   </property>
   <property name="maximum">
    <number>65535</number>
   </property>
   <property name="value">
    <number>100</number>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>420</y>
     <width>91</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Modbus  adr</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>420</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Register</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBox_Reg">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>440</y>
     <width>91</width>
     <height>27</height>
    </rect>
   </property>
   <property name="maximum">
    <number>65535</number>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>420</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Value</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBox_Val">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>440</y>
     <width>91</width>
     <height>27</height>
    </rect>
   </property>
   <property name="maximum">
    <number>65535</number>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_send">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>440</y>
     <width>99</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>send</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_port">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>440</y>
     <width>113</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>/dev/ttyUSB0</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>420</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Port</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_help">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>470</y>
     <width>711</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>help</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_con">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>410</y>
     <width>71</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Connect</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_discon">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>410</y>
     <width>71</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>DisConnect</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_start">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>400</y>
     <width>61</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Start</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_stop">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>400</y>
     <width>61</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Stop</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
